frame/system/src/lib.rs

```diff
@@ -1483,9 +1497,11 @@ impl<T: Trait + Send + Sync> CheckWeight<T> where
 	fn get_priority(info: &DispatchInfoOf<T::Call>) -> TransactionPriority {
 		match info.class {
 			DispatchClass::Normal => info.weight.into(),
```

Comment:
Just tagging the related issue to this funny line: https://github.com/paritytech/substrate/issues/5672

Probably not a problem that needs to be solved now